doc/src/docs/InputOutputReference/01d-InputOutputReference.md

```diff
@@ -17236,6 +17247,8 @@ Note that the key value for these outputs is the AirLoopHVAC name, not the name
 
 * HVAC,Average,Air System Mixed Air Mass Flow Rate [kg/s]
 
+* HVAC,Average,Air System Ourdoor Air Maximum Flow Fraction []
```

Comment:
Good catch. I think I corrected code and forgot to make correction in IO Ref. Thanks.